Makefile

```diff
@@ -89,11 +89,11 @@ run: install-local generate fmt vet ## Run a controller from your host, proxying
 
 .PHONY: docker-build
 docker-build: ## Build docker image with the manager.
-	@eval $$(minikube docker-env --profile kgw); DOCKER_BUILDKIT=1 docker build -t ${IMG} .
+	@eval $$(SHELL=/bin/bash minikube docker-env --profile kgw); DOCKER_BUILDKIT=1 docker build -t ${IMG} .
```

Comment:
We can just set this SHELL var at the top in Makefile

Reply:
this is just a temporary workaround for a bug in minikube shell - it's doing shell detection wrong and we'll just remove this once it's fixed.